polaris-core/src/main/java/org/apache/polaris/core/entity/IcebergTableLikeEntity.java

```diff
@@ -24,7 +24,7 @@
 import org.apache.iceberg.catalog.TableIdentifier;
 import org.apache.iceberg.rest.RESTUtil;
 
-public class TableLikeEntity extends PolarisEntity {
+public class IcebergTableLikeEntity extends PolarisEntity {
   // For applicable types, this key on the "internalProperties" map will return the location
   // of the internalProperties JSON file.
   public static final String METADATA_LOCATION_KEY = "metadata-location";
```

Comment:
This is the reason that we don't just make GENERIC a subType of TABLE_LIKE -- TABLE_LIKE assumes a metadata location (among other things), and we probably don't want to bake that assumption into generic tables. Beyond that, this leads us to refactors in resolution that should set the stage for volumes which would have a similar problem as generic tables.